File: IronVault/UI/ironvault_mobile/lib/models/tracker.dart


import 'package:ironvault_mobile/models/korisnik.dart';

import 'package:json_annotation/json_annotation.dart';

part 'tracker.g.dart';

@JsonSerializable()
class Tracker {
  int? korisnikId;
  Korisnik? korisnik;

  int? ukupnoKupljeno;
  int? ukupnoPrisutava;


  Tracker();

  factory Tracker.fromJson(Map<String, dynamic> json) =>
      _$TrackerFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$TrackerToJson(this);
}

File: IronVault/IronVault.Model/Models/Tracker.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public class Tracker
    {

        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }
        public float? ukupnoKupljeno { get; set; }
        public int? ukupnoPrisutava { get; set; }
    }
}
